dd-sdk-android-ndk/src/main/cpp/utils/backtrace-handler.cpp

```diff
@@ -56,10 +56,17 @@ namespace {
         return std::string(address_as_hexa);
     }
 
-    void get_info_from_address(size_t index, const uintptr_t address, std::string *backtrace) {
-        backtrace->append(std::to_string(index));
+    void get_info_from_address(size_t index,
+                               size_t start_appending_from_frame_index,
```

Comment:
The name of this variable isn't really meaningful, and a variable should be an action `start_appending_from_frame_index`. Why don't you use a more explicit name aka `stack_frames_start_index`, or just `start_index`

Reply:
For me it was explicit enough saying that we are starting unwinding from frame index 0 but we start appending from `start_appending_from_frame_index`